etc/builtin.conf

```diff
@@ -97,3 +97,15 @@ sub-shadow-offset=4
 [box]
 profile=osd-box
 profile=sub-box
+
+[image]
+profile-cond=(get('current-tracks/video', {}).image and not get('current-tracks/video', {}).albumart) or (not get('current-tracks/video') and get('user-data/mpv/image'))
+profile-restore=copy
+input-commands=no-osd set user-data/mpv/image 1; enable-section image allow-hide-cursor+allow-vo-dragging
+osc=no
```

Comment:
I don't think this should be disabled.

Reply:
osc completely breaks drag to pan from the bottom half of the window. We can also use script-opt=osc-visibility=never if preferred.

Reply:
osc is useful for images, to see a title, playlist position, navigate, pause. I don't think we should make some other program with those settings, it still should be mpv.

Reply:
As you wish but then drag to pan is completely broken (it would be nice if it worked like uosc). I didn't even understand why it wasn't working at first. Do you want to use `--script-opt=osc-deadzonesize=0.9` at least?